### DISCUSSION TITLE
Ordering of widths in snapshots.yml not preserved

### DISCUSSION THREAD

**Discussion by cancan101** [percy/cli#744] [Ideas]
I am looking to explicitly specify the order of the widths that I want the snapshot tests to run over but it looks like the order that I specify in the `snapshots.yml` is lost and the widths are just sorted from smallest to largest. 

This is problematic as I do want to run on the larger width first as the wider versions of the page I am testing request a superset of the images that are shown on the narrower versions. I can work around this with some extra executes + scrolling but that shouldn't be needed if I can control the order of the widths.

**Comment by jcharnley** [comment]
have u been able to specific multiple heights? I can define as many widths as I like but in percy.yml i can only set height are one variable

minHeight
